README.md

```diff
@@ -104,6 +114,32 @@ Host your own Chocolatey package repository
 
 #### Parameters
 
+##### `chocolatey_server_app_pool_name`
+Set apppool name used by the chocolatey.server website. Defaults to
+'chocolatey.server'.
+
+##### `compatibility_mode`
+Compatibility mode was added to allow this module to be backwards
+compatible with older implementations. Defaults to 'true'.
+
+##### `disable_default_website`
+The default website is stopped to prevent conflicts with the
+chocolatey server website. An alternate port can be defined and
+this option can be set to disabled so it doesn't modify the default
+website. Defaults to 'true'.
+
+##### `default_website_path`
+The default website path is needed in order to modify the default
+website. Defaults to 'C:\inetpub\wwwroot'.
+
+##### `packages_folder`
+An alternate folder can be defined for the .nupkg files.
```

Comment:
I didn't see my previous comment for some reason. From my understanding, server_package_source only specifies where the chocolatey.server nuget package can be found. This setting allows you to change the path of where the nuget packages are served from instead of the default C:\tools\chocolatey.server\App_Data\Packages.